README.md

```diff
@@ -91,19 +91,60 @@ Query Editor unlocks all possibilities of CQL including Used-Defined Functions,
 Example (using the sample table from the Query Configurator case):
 
 ```
-SELECT sensor_id, CAST(temperature as double), registered_at FROM test.test WHERE sensor_id IN (99051fe9-6a9c-46c2-b949-38ef78858dd1, 99051fe9-6a9c-46c2-b949-38ef78858dd0) AND registered_at > $__timeFrom and registered_at < $__timeTo
+SELECT sensor_id, temperature, registered_at, location FROM test.test WHERE sensor_id IN (99051fe9-6a9c-46c2-b949-38ef78858dd1, 99051fe9-6a9c-46c2-b949-38ef78858dd0) AND registered_at > $__timeFrom and registered_at < $__timeTo
 ```
 
-1. Follow the order of the SELECT expressions, it's important! 
+1. Order of fields in the SELECT expression doesn't matter except `ID` field. This field used to distinguish different time series, so it is important to keep it on the first position.
```

Comment:
Well, it is flexible by grafana design, at least I see it that way. Grafana itself doesn't have restrictions such as field order or type and doesn't consider flexibility dangerous. Moreover, now we can allow to use wide range of cassandra types without need to convert them to float or string - and it will just work. Additional fields could be used in table panels or as metadata to template legend.